features/26-mfa/api.md

```diff
@@ -336,6 +336,10 @@ No input is required.
 
 No response body.
 
+### Note
```

Comment:
I assume if MFA is required/optional, this operation would still be denied if not yet authenticated with MFA?